homework3/semi-uber/Server/api/handlers/user.handlers/userDeteleHandler.js

```diff
@@ -0,0 +1,22 @@
+const log4js = require("log4js");
+const logger = log4js.getLogger();
+const { User } = require("../../Schemas/user.schema");
+const errorHandler = require("../error.handler");
+
+async function userDeleteHandler(req, res) {
```

Comment:
You do not need to write `async`, if you are not using `await` in the function

Reply:
done